### PR TITLE
[stable/hoppscotch]: correct hoppscotch ingress object

### PR DESCRIPTION
**Pull request by ChiefAlexander**
## Description

The hoppscotch charts ingress object was not working as intended.

## Checklist

- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)
- [ ] Github actions are passing

### CODE REVIEW

File: stable/hoppscotch/README.md [hunk 1]
```diff
@@ -1,42 +1,16 @@
...
-helm install my-release deliveryhero/hoppscotch
-```
-
-To install with some set values:
-
-```console
-helm install my-release deliveryhero/hoppscotch --set values_key1=value1 --set values_key2=value2
-```
-
-To install with custom values file:
```
Comment (max-rocket-internet) [inline stable/hoppscotch/README.md:35]: Can you run the docker command to generate the readme so it doesn't remove these instructions?

Details here: https://github.com/deliveryhero/helm-charts#contributing

File: stable/hoppscotch/templates/ingress.yaml [hunk 2]
```diff
@@ -28,11 +28,11 @@ spec:
   - host: {{ .host | quote }}
     http:
       paths:
-      {{- range .paths }}
-      - path: {{ .path }}
-        backend:
-          serviceName: {{ .backend.serviceName }}
-          servicePort: {{ .backend.servicePort }}
-      {{- end }}
+        {{- range .paths }}
```
Comment (max-rocket-internet) [inline stable/hoppscotch/templates/ingress.yaml:31]: Thanks 🙏

**Review by max-rocket-internet — CHANGES_REQUESTED** [review]

File: stable/hoppscotch/README.md (hunk 1 above)
Comment (ChiefAlexander) [reply stable/hoppscotch/README.md:35]: I did attempt to run this and commit the result. I'll try again.

Comment (ChiefAlexander) [reply stable/hoppscotch/README.md]: Pretty sure I ran it within the wrong directory the first time 🤦 .

**Review by max-rocket-internet — APPROVED** [review]

**Comment by max-rocket-internet** [comment]
Thank you @ChiefAlexander 💚

**Pull request merged**